cubes/sql/expressions.py

```diff
@@ -215,7 +214,11 @@ def compile_unary(self, context, operator, operand):
 
         return result
 
+    def compile_if_else(self, context, test, true_expression, false_expression):
```

Comment:
This should rather be a global function, as it does not access any of the compiler's instance variables. We don't need to pass the context either, just use the necessary three arguments.